src/test/config.sol

```diff
@@ -125,11 +125,11 @@ contract Config {
             cat_box:               20 * MILLION,            // In whole Dai units
             dog_Hole:              70 * MILLION,            // In whole Dai units
             esm_min:               150 * THOUSAND,          // In whole MKR units
-            pause_authority:       chief,                   // Pause authority
-            osm_mom_authority:     chief,                   // OsmMom authority
-            flipper_mom_authority: chief,                   // FlipperMom authority
-            clipper_mom_authority: chief,                   // ClipperMom authority
-            ilk_count:             54                       // Num expected in system
+            pause_authority:       chief,                 // Pause authority
+            osm_mom_authority:     chief,                 // OsmMom authority
+            flipper_mom_authority: chief,                 // FlipperMom authority
+            clipper_mom_authority: chief,                 // ClipperMom authority
```

Comment:
Should probably fix the spacing here to avoid it from being in the diff.
```suggestion
            pause_authority:       chief,                   // Pause authority
            osm_mom_authority:     chief,                   // OsmMom authority
            flipper_mom_authority: chief,                   // FlipperMom authority
            clipper_mom_authority: chief,                   // ClipperMom authority
```